apps/backoffice-v2/src/lib/blocks/variants/KybDemoBlocks/hooks/useKybDemoBlocksLogic/useKybDemoBlocksLogic.tsx

```diff
@@ -0,0 +1,294 @@
+import { useParams } from 'react-router-dom';
```

Comment:
Same comment here as in `useDefaultBlocksLogic.tsx`, I really think that if we manage to minimize and better control the code in this files then we'd be able to manage them much more easily.